verification/src/block_verifier.rs

```diff
@@ -407,7 +389,8 @@ impl TransactionsVerifier {
             })
             .collect::<Result<Vec<_>, _>>()?;
 
-        let sum: Cycle = cycles_set.iter().sum();
+        // excludes cellbase cycle
+        let sum: Cycle = cycles_set.iter().skip(1).sum();
```

Comment:
I think we need to exclude the cycles of cellbase in block assembler. For example, the limit is 1000, and cellbase has used 200, then the block assembler can only fetch transactions from pool up to 800 cycles.

Reply:
I don't know how to implement this, because we need to fetch transactions with cycle limit before creating a cellbase:
https://github.com/nervosnetwork/ckb/blob/960f0c9059ab579d6e5ca3becd762fb278968cb4/miner/src/block_assembler.rs#L361

Reply:
Let's revert it first, include cellbase in the limit. Add a comment in `BlockAssembler::get_block_template` that

```
// It is assumed that cellbase transaction consumes 0 cycles, so it is not excluded when getting transactions from pool.
```
```